km_api/know_me/models.py

```diff
@@ -266,6 +267,43 @@ def get_absolute_url(self):
         """
         return reverse('know-me:km-user-detail', kwargs={'pk': self.pk})
 
+    def get_emergency_contact_detail_url(self, request=None):
```

Comment:
This method doesn't make sense here. First of all there's no reference to the specific emergency item, so the returned URL is incorrect. Secondly, this is covered by `EmergencyContact.get_absolute_url()`.